MeteoSwiss-APN/dawn
# Clang format

## Technical Description

- Adds a pre-commit hook that runs clang-format on any changed files
- Removes "make format" because this yoda-target does not what we want it should do
- Adds a `clang-format_all.sh` script in `scripts` (could be turned into a cmake target later)
- `storage_dsl.hpp` is the only file that is not only reformatted because comments were not handled properly by clang-format.

### Testing / Notes to the reviewer

- Run the clang-format script with your version (6.0!) and check that nothing changes
- Check the files which have been changed whether they are internal or external (i.e. check ignore list)
